_summaries/touvron2020training_2.md

```diff
@@ -3,17 +3,56 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # kelseyball
+score: # 6
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Main contributions
+- They efficiently train a vision transformer on only Imagenet1k (trains in a few days on one machine) by distilling with a convnet teacher model
+- They introduce a new distillation procedure and show that it works better than the usual methods
+
+# Their model: Data-efficient image Transformers (DeiT)
+- Architecture is identical to Dosovitskiy et al 2015; differences is the distillation strategy
+- Larger model is the same as ViT-Base; smaller ones have fewer params and faster throughput
+
+<img width="500px" src="touvron2020training_2a.png"/>
+
+- Training procedure: train on Imagenet1k, finetune on same images at larger resolution
+
+# New distillation method: "Hard-Label Distillation"
```

Comment:
I think a brief review of what Vision Transformers could be helpful, if only for completion.

More importantly, I believe distillation is a new concept for us so do explain what it is.